Sources/GryphonLib/KotlinTranslator.swift

```diff
@@ -1009,8 +1009,12 @@ public class KotlinTranslator {
 				result.append(" -> ")
 			}
 
-			if switchCase.statements.count == 1,
-				let onlyStatement = switchCase.statements.first
+			let statements = switchCase.statements.prefix {
+					!($0 is BreakStatement)
+				}
+
+			if statements.count == 1,
+				let onlyStatement = statements.first
```

Comment:
I think we could improve this by moving it from the `KotlinTranslator.swift` file to the `TranspilationPass.swift` file. Specifically, there's already a `RemoveBreaksInSwitchesTranspilationPass` class in there with a `removeBreaksInSwitchCase` method that might be a good fit for your changes.

This is just to make it consistent with the rest of the codebase: in general, all changes to the input code (like removing or adding statements) are made in the `TranspilationPasses`, so that the AST is ready to just be turned into text when it gets to the `KotlinTranslator`.

Reply:
Sounds good. I'll look into it. Thanks for the feedback.

Reply:
Updated!